src/css/app.sass

```diff
@@ -4,20 +4,20 @@
 
 .small-screen-only
     @media ( max-width:  $breakpoint-xs-max)
-        display: block  
+        display: block
     @media  ( min-width:  $breakpoint-sm-min)
         display: none
 
 .large-screen-only
     @media ( max-width:  $breakpoint-xs-max)
         display: none
     @media ( min-width:  $breakpoint-sm-min)
-        display: block       
+        display: block
 
 .constrain
     max-width: 975px
     margin: 0 auto
 
 .constrain-more
```

Comment:
1) src/css/app.sass 
styles 4 spaces
![image](https://user-images.githubusercontent.com/59572810/221367125-51dfc23b-5714-4d83-b3a7-a32841faed2f.png)


2) src/pages/PageHome.vue
styles 1 spaace 
![image](https://user-images.githubusercontent.com/59572810/221367172-c17dda83-97b5-4228-9b86-9d599b8b122e.png)


USE THE SAME SPACES, USE PRETTIER, use 2 spaces

NOT A MISTAKE, but it is more common to use brackets like this:


.test-lcass {
  cursor: pointer;
}